README.md

```diff
@@ -10,9 +10,16 @@ Ballerina MySQL Library
 This library provides the functionality required to access and manipulate data stored in a MySQL database.
 
 ### Prerequisite
-Add the MySQL driver JAR as a native library dependency in your Ballerina project's `Ballerina.toml` file.
-It is recommended to use a MySQL driver version greater than 8.0.13 as this library uses the database properties
-from the MySQL driver version 8.0.13 onwards.
+It is required to import the MySQL driver dependency in order to connect to a MySQL database. The `ballerinax/mysql.driver`
```

Comment:
Add the MySQL driver as a dependency to the ballerina project.

>> Note: `ballerinax/mysql` supports MySQL drivers above 8.0.13.

This can be easily achieved by importing `ballerinax/mysql.driver` module,
 ```ballerina
 import ballerinax/mysql.driver as _;
 ```

`ballerinax/mysql.driver` package bundles the latest MySQL driver JAR.

>> Tip: Graalvm native build is supported when `ballerinax/mysql` is used along with `ballerinax/mysql.driver`

In any case, if the MySQL driver of a specific version needs to be imported, you can add it as a dependency in Ballerina.toml.

```
// Give both methods this can be achieved. Or it is better if we can point to the common doc to import java libs to ballerina
```

Reply:
Fixed in [a9ff096](https://github.com/ballerina-platform/module-ballerinax-mysql/pull/726/commits/a9ff0969f27883d6d959dead62ab704c7e893fa3)